Tools/2MGFX/Program.cs

```diff
@@ -7,6 +7,12 @@ public static class Program
     {
         public static int Main(string[] args)
         {
+            if (!Environment.Is64BitOperatingSystem)
```

Comment:
Would this be safer as `Environment.Is64BitProcess`? 

https://msdn.microsoft.com/en-us/library/system.environment.is64bitprocess(v=vs.110).aspx